src/components/Activity.tsx

```diff
@@ -1,15 +1,24 @@
 import ActivityCard from './ActivityCard'
 import { useAllActivities } from '../hooks/useAllActivities';
+import { CircularProgress } from '@mui/material';
 
 
```

Comment:
best to avoild console.log() in production code

Reply:
Fixed.